.eslintrc

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Two comments here: 

1. The config file name should be `.estlintrc.json`. From Eslint docs: 

```
ESLint supports configuration files in several formats:

    JavaScript - use .eslintrc.js and export an object containing your configuration.
    ...
    JSON - use .eslintrc.json to define the configuration structure. ESLint's JSON files also allow JavaScript-style comments.
    Deprecated - use .eslintrc, which can be either JSON or YAML.
```

2. When we run `npm run lint` from the project root then ESlint finds `./web/.eslintrc.json`. We need to discuss whether it makes sense to have an additional `.eslintrc.json` file at the project root.